packages/components/combobox/src/combobox.stories.ts

```diff
@@ -0,0 +1,329 @@
+      <sl-listbox>${Array.from({ length: 5000 }).map((_, i) => html`<sl-option>Option ${i}</sl-option>`)}</sl-listbox>
+    `
+  }
+};
+
+export const All: Story = {
+  render: () => html`
+    <style>
+      .container {
+        align-items: center;
+        display: grid;
+        grid-template-columns: minmax(auto, 350px) minmax(auto, 350px);
+        gap: 1rem;
+
+        span {
+          justify-self: center;
+        }
+      }
+    </style>
+    <div class="container">
+      <span>md</span>
+      <span>lg</span>
+      <sl-combobox placeholder="Empty">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+      <sl-combobox placeholder="Empty" size="lg">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+
+      <sl-combobox placeholder="Selected" value="Option 2">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+      <sl-combobox placeholder="Selected" size="lg" value="Option 2">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+
+      <sl-combobox readonly placeholder="Readonly">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+      <sl-combobox readonly placeholder="Readonly" size="lg">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+
+      <sl-combobox disabled placeholder="Disabled">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+      <sl-combobox disabled placeholder="Disabled" size="lg">
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+
+      <sl-combobox disabled multiple placeholder="Multiple, disabled" .value=${['Option 1', 'Option 2']}>
+        <sl-listbox>
+          <sl-option>Option 1</sl-option>
+          <sl-option>Option 2</sl-option>
+          <sl-option>Option 3</sl-option>
+        </sl-listbox>
+      </sl-combobox>
+      <sl-combobox disabled multiple placeholder="Multiple, disabled" size="lg" .value=${['Option 1', 'Option 2']}>
```

Comment:
In the multiple variant and disabled, the first tag can be focused
<img width="1140" alt="image" src="https://github.com/user-attachments/assets/4176c3e0-ff77-4b44-bcdf-05e5da54b9fa">

Reply:
Fixed